mmv1/products/compute/RegionResizeRequest.yaml

```diff
@@ -0,0 +1,413 @@
+- name: 'status'
+  type: NestedObject
+  description: |
+    [Output only] Status of the request.
+  output: true
+  properties:
+  # Status.error
+  - name: 'error'
+    type: NestedObject
+    description: |
+      [Output only] Fatal errors encountered during the queueing or provisioning phases of the ResizeRequest that caused the transition to the FAILED state. Contrary to the lastAttempt errors, this field is final and errors are never removed from here, as the ResizeRequest is not going to retry.
+    output: true
+    properties:
+    - name: 'errors'
+      type: NestedObject
+      description: |
+        [Output Only] The array of errors encountered while processing this operation.
+      output: true
+      properties:
+      - name: 'code'
+        type: String
+        description: |
+          [Output Only] The error type identifier for this error.
+        output: true
+      - name: 'location'
+        type: String
+        description: |
+          [Output Only] Indicates the field in the request that caused the error. This property is optional.
+        output: true
+      - name: 'message'
+        type: String
+        description: |
+          [Output Only] An optional, human-readable error message.
+        output: true
+      - name: 'errorDetails'
+        type: NestedObject
+        description: |
+          [Output Only] An optional list of messages that contain the error details. There is a set of defined message types to use for providing details.The syntax depends on the error code. For example, QuotaExceededInfo will have details when the error code is QUOTA_EXCEEDED.
+        output: true
+        properties:
+        - name: 'errorInfo'
+          type: NestedObject
+          description: |
+            [Output Only]
+          output: true
+          properties:
+          - name: 'reason'
+            type: String
+            description: |
+              The reason of the error. This is a constant value that identifies the proximate cause of the error. Error reasons are unique within a particular domain of errors. This should be at most 63 characters and match a regular expression of [A-Z][A-Z0-9_]+[A-Z0-9], which represents UPPER_SNAKE_CASE.
+            output: true
+          - name: 'domain'
+            type: String
+            description: |
+              The logical grouping to which the "reason" belongs. The error domain is typically the registered service name of the tool or product that generates the error. Example: "pubsub.googleapis.com". If the error is generated by some common infrastructure, the error domain must be a globally unique value that identifies the infrastructure. For Google API infrastructure, the error domain is "googleapis.com".
+            output: true
+          - name: 'metadatas'
```

Comment:
is the name really "metadatas"? I thought the plural is "metadata"

Reply:
As [rileykarson](https://github.com/rileykarson) mentioned, in api it shows [metadatas](https://screenshot.googleplex.com/8SGmaXNZdbGWCmP). On the other side in mixer, it named as [metadata](https://screenshot.googleplex.com/3ReGSjvFuJJQXyG). Which one should we follow here?

Reply:
OK, I'm simply surprised. Probably that's not something we can cahnge (and not sth we should change after it's been published).
Please consider this resolved